lib/model/helpers/CentrelineFeature.js

```diff
@@ -0,0 +1,10 @@
+import { CentrelineType } from '@/lib/Constants';
+import Joi from '@/lib/model/Joi';
+
+export default {
```

Comment:
one issue here is that we're conflating models for handling database persistence (e.g. `StudyRequest`) with other validation schemas, such as for endpoint query / response types.  we may need a separate space for the latter (e.g. `@/lib/schemas`, or some additional thought into how to bring these two use cases for `Joi` validation together.

Reply:
the main difference between this and `GeoJson.Point` is that the latter is self-contained within a `Joi.object()` declaration, and is intended for use as the schema for individual fields within larger objects.  By contrast, this schema is more of a "mixin": it contains two fields, which are sometimes used standalone (e.g. in various endpoint query schemas) and sometimes used alongside other fields (e.g. in `StudyRequest`).

let's leave it for now, but we might (as mentioned above) eventually want to move it.